src/message.rs

```diff
@@ -760,6 +763,11 @@ impl Message {
             .await
             .is_ok()
     }
+
+    /// Get the error of the message.
```

Comment:
While we are at fixing API style, according to https://github.com/rust-lang/rfcs/blob/master/text/0505-api-comment-conventions.md this should be "Gets" or even "Returns", not "Get":
> The summary line should be written in third person singular present indicative form. Basically, this means write "Returns" instead of "Return".

Standard library follows this style.

Reply:
Alright, I didn't know that document before :)